### DISCUSSION TITLE
part-ii/2020-86/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#238] [Question comments]
Part II 2020 Logic and Set Theory: Paper 1, Section II, H 

https://questions.tripos.org/part-ii/2020-86/

**Comment by dropdbassett** [comment]
<details><summary>Part (i)</summary>

False; given any $\lambda$, take $\kappa = \mathrm{card}\left(\gamma(2^\lambda)\right)$. 
</details>

<details><summary>Part (ii)</summary>

True; note $2^\lambda = 2^{\lambda \lambda} = \left(2^\lambda\right)^\lambda$. But $2^\lambda > \lambda \ge \kappa > 2$, and hence $2^\lambda = \left(2^\lambda\right)^\lambda \ge \kappa^\lambda \ge 2^\lambda$.
</details>

<details><summary>Part (iii)</summary>

False; let $\lambda = 2^{\aleph_0}$ and $\kappa = \aleph_0$. Then $\lambda^\kappa = \lambda$, and $\kappa^\lambda = 2^\lambda > \lambda$.
</details>
